.versions

```diff
@@ -1,11 +1,11 @@
 allow-deny@1.1.0
```

Comment:
Don't modify this file. It is modified when I publish the package.

Reply:
yes sure i will revert my changes.